core/modules/notification/store/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Module } from 'vuex'
+import NotificationItem from '../types/NotificationItem'
+
+export const module: Module<any, any> = {
```

Comment:
you can create NotificationState object in /types to  match `Module<>` first 'argument'